src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/ReclusterServiceIntTest.kt

```diff
@@ -35,31 +29,38 @@ class ReclusterServiceIntTest : MessagingMultiNodeTestBase() {
 
     @Test
     fun `should add a created record to a cluster if it is set to need attention`() {
-      val personA = createPerson(createRandomPersonDetails())
-      createPersonKey(status = UUIDStatusType.NEEDS_ATTENTION)
+      val personA = createPerson(createRandomProbationPersonDetails())
+      val cluster = createPersonKey(status = UUIDStatusType.NEEDS_ATTENTION)
         .addPerson(personA)
 
       stubPersonMatchUpsert()
       stubOnePersonMatchHighConfidenceMatch(matchedRecord = personA.matchId)
-      val cId = randomCId()
-      publishLibraMessage(libraHearing(firstName = randomName(), lastName = randomName(), cId = cId, dateOfBirth = randomDate().format(DateTimeFormatter.ofPattern("dd/MM/yyyy")), cro = "", pncNumber = randomPnc(), postcode = randomPostcode()))
+
+      val newPersonCrn = randomCrn()
+      probationEventAndResponseSetup(OFFENDER_ALIAS_CHANGED, ApiResponseSetup(crn = newPersonCrn))
```

Comment:
I know that the type of event published here is not directly relevant to the test and that the outcome does not depend on it.

`OFFENDER_ALIAS_CHANGED` is the last of the database events, and will be got rid of quite soon hopefully, which will necessitate a change here
That will mean examining this code and working out whether it has anything to do with aliases. We could save ourselves the brainpower by using


```
      probationDomainEventAndResponseSetup(eventType = NEW_OFFENDER_CREATED, ApiResponseSetup(crn = newPersonCrn)))
      ```